### PR TITLE
Bump the fq_driver default backlog to 100000

### PR DESCRIPTION
**Pull request by postwait**

### CODE REVIEW

File: docs/config/modules/opentsdb.xml [hunk 1]
```diff
@@ -78,14 +78,28 @@
...
+              Host           "localhost"
+              Port           "4242"
+              HostTags       "datacenter=east hosttype=backend"
+              StoreRates     false
+              AlwaysAppendDS false
+            &lt;/Node&gt;
+          &lt;/Plugin&gt;
+        </code>
+      </para>
+      <para>The collectd write_tsdb plugin config example above will direct metric data to the broker:4242 for ingestion</para>
```
Comment (esproul) [inline docs/config/modules/opentsdb.xml]: Should be "opentsdb" rather than "collectd"?

**Comment by esproul** [comment]
Are the changes to opentsdb.xml, test_ingestor.xml and noit_check_tools_h.xml deliberately part of this change? They don't look related.

**Comment by postwait** [comment]
they are artifacts of the docs build.

File: docs/config/modules/opentsdb.xml (hunk 1 above)
Comment (postwait) [reply docs/config/modules/opentsdb.xml]: yes. a bug for @vasuraman11

**Comment by esproul** [comment]
> they are artifacts of the docs build.

Understood, thanks.

**Pull request merged**